kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -751,6 +751,57 @@ object KyuubiConf {
       .booleanConf
       .createWithDefault(true)
 
+  //------------------------Jetty Server-------------------------
+  val FRONTEND_JETTYSERVER_USE_SSL: ConfigEntry[Boolean] =
+    buildConf("kyuubi.frontend.jettyServer.use.SSL")
```

Comment:
considering there are existing configurations `kyuubi.frontend.rest.bind.host` and `kyuubi.frontend.rest.bind.port`, let's put those configurations under namespace `kyuubi.frontend.rest.*`